src/schema/types/mutations/User.ts

```diff
@@ -1,41 +1,79 @@
-import { arg, nonNull } from "nexus";
+import { arg, nonNull, stringArg } from "nexus";
+
+import { UserService } from "service";
+import { UserCreateArgs, UserUpdateArgs } from "interface/User";
+import { checkAuthCode, sendAuthCode } from "util/emailAuth";
+
+export const login = {
+    args: {
+        email: nonNull(stringArg()),
+        password: nonNull(stringArg()),
+    },
+    resolve: async (_: any, args: any, ctx: any) => {
```

Comment:
```suggestion
    resolve: async (_: any, args: any, __ : any) => {
```

`ctx`를 사용하지 않으니, `__`로 바꾸는 것이 좋지 않을까 생각됩니다.